File: CS/WindowsFormsApplication1/ChildForm2.cs
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace DragDataBetweenMdiTabbedForms
{
	public partial class ChildForm2 : XtraForm
	{
		public ChildForm2()
		{
			InitializeComponent();
		}

		private void listBoxControl1_DragDrop(object sender, DragEventArgs e)
		{
			if ( e.Data.GetDataPresent(typeof(string)) )
			{
				string str = (string)e.Data.GetData(typeof(string));
				listBoxControl1.Items.Add(str);
			}
		}

		private void listBoxControl1_DragOver(object sender, DragEventArgs e)
		{
			e.Effect = DragDropEffects.Copy;
		}
	}
}

File: VB/WindowsFormsApplication1/ChildForm2.vb
Imports Microsoft.VisualBasic
Imports System.Windows.Forms
Imports DevExpress.XtraEditors

Namespace DragDataBetweenMdiTabbedForms
	Partial Public Class ChildForm2
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub listBoxControl1_DragDrop(ByVal sender As Object, ByVal e As DragEventArgs) Handles listBoxControl1.DragDrop
			If e.Data.GetDataPresent(GetType(String)) Then
				Dim str As String = CStr(e.Data.GetData(GetType(String)))
				listBoxControl1.Items.Add(str)
			End If
		End Sub

		Private Sub listBoxControl1_DragOver(ByVal sender As Object, ByVal e As DragEventArgs) Handles listBoxControl1.DragOver
			e.Effect = DragDropEffects.Copy
		End Sub
	End Class
End Namespace